tests/ffi/tests.cc

```diff
@@ -841,6 +841,12 @@ extern "C" const char *cxx_run_test() noexcept {
   ASSERT(cstr == "foo");
   ASSERT(other_cstr == "test");
 
+  const char* utf8_literal = u8"Test string";
+  const char16_t* utf16_literal  = u"Test string";
```

Comment:
```suggestion
  const char16_t* utf16_literal = u"Test string";
```

Reply:
Sorry about that one, forgot to run clang-format